src/iotjs_util.cpp

```diff
@@ -45,19 +65,19 @@ char* ReadFile(const char* path) {
 }
 
 
-char* AllocBuffer(size_t size) {
-  char* buff = static_cast<char*>(malloc(size));
+jschar* AllocBuffer(size_t size) {
```

Comment:
`AllocBuffer` is not for creating string buffer, it is for creating byte buffer(creating `size` of octets). Thus changed prototype is not correctly describe the purpose of this function.